src/node/routes/login.ts

```diff
@@ -84,6 +89,12 @@ router.post("/", async (req, res) => {
       return redirect(req, res, to, { to: undefined })
     }
 
+    // Note: successful logins should not count against the RateLimiter
+    // which is why this logic must come after the successful login logic
+    if (!limiter.try()) {
```

Comment:
Should we move this after the `console.error` so we can log the failed attempt?

Reply:
Actually maybe we don't want to throw here because the actual error is that the login attempt failed. Then on the next attempt it'll be rate limited.